lib/services/automationManagement/lib/models/index.d.ts

```diff
@@ -2150,11 +2328,12 @@ export interface WebhookCreateOrUpdateParameters {
  * @member {date} [lastModifiedTime] Gets or sets the last modified time.
  * @member {string} [description] Gets or sets the description.
  */
-export interface Webhook {
-  id?: string;
-  name?: string;
-  isEnabled?: boolean;
-  uri?: string;
+export interface Webhook extends BaseResource {
+  readonly id?: string;
+  readonly name?: string;
```

Comment:
Changing an interface's properties to readonly is a breaking change and requires a major version increment.